csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3930,7 +3938,7 @@ The relevant paths for this test are:
 
 #### 6.1.27.8 Vulnerability ID
 
-For each item in `/vulnerabilities` it must be tested that at least one of the elements `cve` or `id` is present.
+For each item in `/vulnerabilities` it must be tested that at least one of the elements `cve` or `ids` is present.
```

Comment:
@sthagen: Do we need to clarify here, that the `ids` MUST NOT be empty?

Reply:
I think cardinality is good as schema constraint - that is what the schema can do. So, better not duplicate. Empty strings do not convey any useful notion of identity.